scripts/openzfs-merge.sh

```diff
@@ -80,12 +80,12 @@ clean_unmerged() {
 	git cherry-pick --abort
 	git checkout master
 	git branch -D "autoport-oz$OPENZFS_ISSUE" > /dev/null
-	rm "$TMP_FILE"
+	rm "$TMP_FILE" > /dev/null
```

Comment:
`rm -f "$TMP_FILE"` is generally a cleaner way to handle this.  It won't generate a warning even if the file is missing.